tests/robot-tests/tests/general_public/release_page_absence.robot

```diff
@@ -63,6 +63,7 @@ Validate "About these statistics" -- "Last updated"
     user closes details dropdown    See all updates (2)
 
 Validate "Useful information"
+     user checks page contains element    link:Download all data
```

Comment:
I think it might be worth running the robot formatter on this file as it looks like the formatting is out here.